Network/netx-getting-started-guide.md

```diff
@@ -0,0 +1,273 @@
+{{{
```

Comment:
does this replace https://www.ctl.io/knowledge-base/network/getting-started-with-netx/ ?  

if so your pull request should delete this old kb.

what about https://www.ctl.io/knowledge-base/network/netx-locations-and-endpoints/ ?